python/lsst/ts/m2gui/log_window_handler.py

```diff
@@ -0,0 +1,45 @@
+# This file is part of ts_m2gui.
+#
+# Developed for the LSST Data Management System.
+# This product includes software developed by the LSST Project
+# (https://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+__all__ = ["LogWindowHandler"]
+
+import logging
+
+
+class LogWindowHandler(logging.Handler):
```

Comment:
I tried this but this can not work in Python for the following error:

```
RuntimeError: '__init__' method of object's base class (WidgetLogWindow) not called.
```

After the googling, I realized maybe the calling of `super().__init__()` in Python is a problem for the multiple inherence if it is required for the parental classes (`logging.Handler` and `QPlainTextEdit`).

1. https://realpython.com/lessons/multiple-inheritance-python/
2. https://stackoverflow.com/questions/34884567/python-multiple-inheritance-passing-arguments-to-constructors-using-super
3. https://realpython.com/python-super/

Thanks!

Reply:
super() changed from Python2 to Python3. For multiple inheritance, if you don't need to pass arguments, one super().__init__() call works. See https://stackoverflow.com/questions/26927571/multiple-inheritance-in-python3-with-different-signatures